ds_judgements_public_ui/sass/includes/_judgment_text_toolbar.scss

```diff
@@ -61,6 +58,8 @@
     @media (min-width: $grid-breakpoint-small) {
       padding-right: $space-8;
     }
+
+    @include link;
```

Comment:
Is there a standard for the order of these things (is there a linter that'd help make this consistent?) 
Includes at the bottom feels odd...